src/AppLovinMAXAdView.js

```diff
@@ -111,6 +111,19 @@ const AdView = (props) => {
     if (props.onAdRevenuePaid) props.onAdRevenuePaid(event.nativeEvent);
   };
 
+  const checkExtraParameter = (name, params) => {
```

Comment:
Since we're mutating the parameters passed in, I would rename function accordingly, and pluralize since we are sanitizing a dictionary of parameter**s**.

```suggestion
  const sanitizeExtraParameters = (name, params) => {
```

Reply:
Updated.